vendor.conf

```diff
@@ -26,7 +26,7 @@ github.com/imdario/mergo                            7c29201646fa3de8506f70121347
 golang.org/x/sync                                   e225da77a7e68af35c70ccbf71af2b83e6acac3c
 
 # buildkit
-github.com/moby/buildkit                            ebcef1f69af0bbca077efa9a960a481e579a0e89 # v0.6.4
+github.com/moby/buildkit                            b26cff2413cc6a466f8739262efa13bd126f8fc7 # v0.6.4 + aa7df97d7136e732561b59f87a38ad52d46d3b19
```

Comment:
@tonistiigi do you know if there's other fixes pending for the v0.6 / docker-19.03 branch? (or of otherwise a new version should be tagged?)

Reply:
Now that v0.7 is out I don't think we plan to do v0.6 releases/tags anymore. Not are of anything else that needs to be picked.